tests/testthat/test-utils.R

```diff
@@ -5,3 +5,49 @@ testthat::test_that("get_bs_version", {
 testthat::test_that("panel_item", {
   testthat::expect_s3_class(panel_item("LABEL", shiny::tags$div()), "shiny.tag")
 })
+
+testthat::test_that("Test to_flextable: supported class", {
```

Comment:
To keep test naming consistent

```suggestion
testthat::test_that("to_flextable supported class", {
```